### ISSUE TITLE
Force Ticket assignment before working on it

### ISSUE THREAD

**Issue by jeffchulg** [osTicket/osTicket-1.7#505]
Hi, my boss asked me if there is the ability to force ticket assignment before being able even to answer it. Maybe via a parameter...

What is your point about it ?

**Comment by protich** [comment]
You can auto-assign using ticket filters or help topics (web tickets).

**Comment by jeffchulg** [comment]
I understand this way of doing, but it's not what he asks : a ticket can be resolved by anybody in a team (assuming they share the same knowledge). So ok, ticket can be assigned to team, but not to a member. What he asks is that if there is no assignement on a ticket, one can't do anything on it except assign to him (maybe at reply)...

**Comment by protich** [comment]
Auto-claim on reply is not a supported feature. That said, when a ticket is closed it gets "assigned" to closing staff.

**Comment by jeffchulg** [comment]
Would the "Auto-claim on reply" feature be implemented then ?

We have at this time tickets that can stay open for months, so it's not enough, to him

**Comment by protich** [comment]
To be honest, it's not a feature on my todo list. May be it's something we can revisit post v1.7 release or perhaps you can implement it yourself and make a pull request?

**Comment by jeffchulg** [comment]
yes, it's not so difficult.. I'd just need time to get into github... 
I'll try it tomorrow

**Comment by Richiricheh** [comment]
It was made an internal policy with my team that if they didn't claim a ticket from the teams queue prior to working on it then it would be treated as a negative item in their Year-end and/or quarterly review. Running an aged ticket report and a separate aged ticket report w/ no assignee + response would not be difficult. The later report could even be run on a weekly or daily basis is that is what is required to keep the team on-board with the policy change.

Since OSTicket automatically assigns the staff member who closes a ticket the only relevant item which would cause concern would be aged tickets with no assignee. If dev time is short a report like this might work instead.
